deploy/002_deploy_optimism_adapter.ts

```diff
@@ -1,15 +1,16 @@
-import { L1_ADDRESS_MAP, USDC, WETH } from "./consts";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { CHAIN_IDs } from "../utils";
+import { L1_ADDRESS_MAP, OP_STACK_ADDRESS_MAP, USDC, WETH } from "./consts";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await hre.getNamedAccounts();
   const chainId = parseInt(await hre.getChainId());
 
   const args = [
     WETH[chainId],
-    L1_ADDRESS_MAP[chainId].optimismCrossDomainMessenger,
-    L1_ADDRESS_MAP[chainId].optimismStandardBridge,
+    OP_STACK_ADDRESS_MAP[chainId][CHAIN_IDs.OPTIMISM].L1CrossDomainMessenger,
```

Comment:
Since `CHAIN_IDs.OPTIMISM` is hardcoded, these scripts will only ever work for testnet. Do you want to do something like `const spokeChainId = chainId === CHAIN_IDs.MAINNET ? CHAIN_IDs.OPTIMISM : CHAIN_IDs.OPTIMISM_SEPOLIA`? (Although this might not work for all chains). I'm mentioning this since you defined the testnet addresses in `OP_STACK_ADDRESS_MAP`, but they won't be used without changing the script. Maybe that's not that big of a deal?